enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Notebook.tsx

```diff
@@ -40,7 +40,8 @@ export const Notebook = ({ onApply = () => {} }: NotebookProps) => {
           question={question}
           isDirty={isDirty}
           isRunnable={isRunnable}
-          isResultDirty={isDirty}
+          // the visualization button relies on this boolean
```

Comment:
The visualization button relies on `isResultDirty` to figure out whether or not the click should register on the button. However, on the SDK, if the user clicks on the Visualize button, it should show them the visualization no matter what. This is all that this prop does